stakingreward/stakingreward.go

```diff
@@ -279,61 +240,37 @@ func (exp *calculator) simulateStakingReward(numberOfDays float64, startingDCRBa
 	return
 }
 
-// commonData grabs the common page data that is available to every page.
-// This is particularly useful for extras.tmpl, parts of which
-// are used on every page
-func (exp *calculator) commonData(r *http.Request) *web.CommonPageData {
-
-	darkMode, err := r.Cookie(web.DarkModeCoookie)
-	if err != nil && err != http.ErrNoCookie {
-		log.Errorf("Cookie dcrdataDarkBG retrieval error: %v", err)
-	}
-	return &web.CommonPageData{
-		Version:       exp.Version,
-		ChainParams:   exp.ChainParams,
-		BlockTimeUnix: int64(exp.ChainParams.TargetTimePerBlock.Seconds()),
-		DevAddress:    exp.pageData.HomeInfo.DevAddress,
-		NetName:       exp.NetName,
-		Links:         web.ExplorerLinks,
-		Cookies: web.Cookies{
-			DarkMode: darkMode != nil && darkMode.Value == "1",
-		},
-		RequestURI: r.URL.RequestURI(),
-		MenuItems:  exp.webServer.MenuItems,
-	}
-}
-
 // stakingReward is the page handler for the "/ticket-reward" path.
-func (ac *calculator) stakingReward(w http.ResponseWriter, r *http.Request) {
-	price := 24.42
-	if ac.xcBot != nil {
-		if rate := ac.xcBot.Conversion(1.0); rate != nil {
+func (calc *calculator) stakingReward(w http.ResponseWriter, r *http.Request) {
+	price := 24.42 // why this value?
```

Comment:
This is a default value to be used whenever `xcBot` is disabled

Reply:
Yes but why `24.42` specifically?

Reply:
Just an arbitrary number